simSPI/tem_distribution_utils.py

```diff
@@ -0,0 +1,52 @@
+"""Utility file for generating and sampling distributions."""
```

Comment:
I like this file!

Reply:
I think it's a good starting point for general stochastic simulators, where the distributional parameters are given, and not every single value.